ui.js

```diff
@@ -0,0 +1,40 @@
+
+const PLAY_ICON = "play_arrow";
+const PAUSE_ICON = "pause";
+
+class UI {
+  constructor(config) {
+    this.game = config.game;
+    this.playButton = config.playButton;
+    this.stepButton = config.stepButton;
+    this.stopButton = config.stopButton;
+    this.generationCounter = config.generationCounter;
+    this.events = config.events;
+    //this.mousePosition = config.mousePosition;
+    this.initUI();
+  }
+
+  initUI() {
+    this.playIcon = this.playButton.firstElementChild;
+    this.stopButton.onclick = () => {
+      this.game.reset();
+    };
+    this.playButton.onclick = () => {
```

Comment:
play button should not be a play button some of the time and a pause button the other part of the time. you called it a play button so it should just be a play button